lib/src/discovery.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "dart:io";
```

Comment:
I know we don't yet have consistent agreement on whether double-vs-single quotes are important enough to standardize in general, but the _vast_ majority, over 97%, of Dart code in google3 at least uses single quotes for imports. I'd be happy to change these in a followup if you prefer not to do it in this PR.

Reply:
I'm a stout holdout for double quotes, so no thanks to changing them.